specs/subresourceintegrity/spec.markdown

```diff
@@ -748,6 +748,21 @@ will likely be difficult to avoid (image's `naturalHeight` and
 `naturalWidth` for instance).
 </section><!-- /Security::cross-origin -->
 
+<section>
+### Behavior of chameleon resources
```

Comment:
I think that paper was focused on sniffing and this section is more about a bunch of bytes having a completely different meaning based on mime confusion.

Although, I think a simpler title like "Impact of out-of-bound data" is much clearer and connects to the broader point we are making in the section.